extra/bunotel/option.go

```diff
@@ -30,3 +32,28 @@ func WithFormattedQueries(format bool) Option {
 		h.formatQueries = format
 	}
 }
+
+// WithTracerProvider returns an Option to use the TracerProvider when
+// creating a Tracer.
+func WithTracerProvider(tp trace.TracerProvider) Option {
+	return func(h *QueryHook) {
+		if tp != nil {
+			h.tracer = tp.Tracer("github.com/uptrace/bun")
+		}
+	}
+}
+
+// WithMeterProvider returns an Option to use the MeterProvider when
+// creating a Meter.
+func WithMeterProvider(mp metric.MeterProvider) Option {
+	return func(h *QueryHook) {
+		if mp != nil {
+			h.meter = mp.Meter("github.com/uptrace/bun")
+			h.queryHistogram, _ = h.meter.Int64Histogram(
```

Comment:
Looks like this will be overridden anyway later when the hooks is created.

Reply:
So, let's just remove this?

Reply:
You are right, I forgot to remove this.